LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -242,34 +242,39 @@ class OnboardingUICoordinator: UINavigationController, CGMManagerOnboarding, Pum
     }
 
     private func constructTherapySettingsViewModel(therapySettings: TherapySettings) -> TherapySettingsViewModel? {
+        return TherapySettingsViewModel(therapySettings: therapySettings, delegate: self)
+    }
+}
+
+extension OnboardingUICoordinator: TherapySettingsViewModelDelegate {
+    func syncBasalRateSchedule(items: [RepeatingScheduleValue<Double>], completion: @escaping (Result<BasalRateSchedule, Error>) -> Void) {
+        // noop
+    }
+    
+    func syncDeliveryLimits(deliveryLimits: DeliveryLimits, completion: @escaping (Result<DeliveryLimits, Error>) -> Void) {
+        // noop
+    }
+    
+    func saveCompletion(for therapySetting: TherapySetting, therapySettings: TherapySettings) {
```

Comment:
[nit] It isn't clear to me what `saveCompletion` means. Perhaps indicating what action is being taking here. Something like `func notifySavedTherapySetting(_ therapySetting: TherapySetting, in therapySettings: TherapySettings)`?

Reply:
This is just the function that used to be of type `SaveCompletion` from `TherapySettingsViewModel`, I'm just using the same name.  And...it is actually intended to "save" the data, not just notify. See https://github.com/tidepool-org/LoopKit/pull/404 for more info.